.golangci.yml

```diff
@@ -2,6 +2,68 @@ run:
   # The default runtime timeout is 1m, which doesn't work well on Github Actions.
   timeout: 4m
 
+linters:
```

Comment:
This has just been moved up with some additional linters added. Its moved up because its one of the most interesting things when wanting to check linting configuration.